src/React/Resources/assets/src/render_controller.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import React, {ReactElement} from 'react';
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    readonly componentValue: string;
+    readonly propsValue: object;
+
+    static values = {
+        component: String,
+        props: Object,
+    };
+
+    connect() {
+        this._dispatchEvent('react:connect', { component: this.componentValue, props: this.propsValue });
+
+        // Use a timeout to avoid mounting and demounting right after
+        if ((this.element as any).timeout) {
```

Comment:
I believe we can use this?

```suggestion
        if ((this.element as HTMLElement).timeout) {
```

And/or could the `timeout` be stored as a property on the component? I don't fully understand the problem, so these suggestions may be nonsense.